source/exe/main_common.cc

```diff
@@ -86,6 +89,8 @@ MainCommonBase::MainCommonBase(const OptionsImpl& options, Event::TimeSystem& ti
                                                          restarter_->logLock());
     break;
   }
+
+  check_platform_settings(file_system);
```

Comment:
checkPlatformSettings(file_system)

Please read over https://github.com/envoyproxy/envoy/blob/master/STYLE.md for conventions.